server/controllers/articles.js

```diff
@@ -121,6 +121,26 @@ class allAboutArticle {
     });
   }
 
+  /**
+   * @author Tuyisenge J.Paul
+   * @param {object} req
+   * @param {object} res
+   * @description This method is used to get all articles
+   */
+  static myArticle(req, res) {
+    const myArticles = articles.filter((article) => {
+      if (article.creatorid === req.user.id) {
+        return true;
+      }
+    });
+    return res.status(200).json({
+      status: '200',
+      message: 'success',
+      myArticles,
+    });
+  }
```

Comment:
do create a helper function which handles all responses..  It is not a good practice to repeatedly write res.status().....  etc